forge-gui/res/cardsfolder/upcoming/everybody_lives!.txt

```diff
@@ -0,0 +1,9 @@
+Name:Everybody Lives!
```

Comment:
is that `!` in the filename ok? It makes me nervous 😅

Reply:
yeah best to remove the ! to be sure